.eslintrc.js

```diff
@@ -39,6 +39,10 @@ module.exports = {
         extensions: [".js", ".jsx", ".ts", ".tsx"],
       },
     },
+    jsdoc: {
+      mode: "typescript",
+      exemptDestructuredRootsFromChecks: true,
+    },
```

Comment:
Based on the doc comments in the PR, none of the destructured roots seem to lack comments, so if we aren't using this setting we can just remove it:
```suggestion
```

Based on the docs, there shouldn't be a need to explicitly set the mode to typescript:

<img width="986" alt="Screenshot 2023-06-08 at 10 15 56 AM" src="https://github.com/evergreen-ci/parsley/assets/47064971/19f93505-75b6-4f01-bd00-5cac0dc262ad">